plugins/hypervisors/kvm/src/com/cloud/hypervisor/kvm/resource/KVMHABase.java

```diff
@@ -34,7 +34,8 @@
     protected static String s_heartBeatPath;
     protected long _heartBeatUpdateTimeout = 60000;
     protected long _heartBeatUpdateFreq = 60000;
-    protected long _heartBeatUpdateMaxRetry = 3;
+    protected long _heartBeatUpdateMaxTries = 5;
```

Comment:
What about externalizing these two information as parameters for operators to customize?